platform-service/src/unstract/platform_service/helper/cost_calculation.py

```diff
@@ -1,16 +1,19 @@
 import json
 import os
-from datetime import datetime, timedelta
+from datetime import datetime, timedelta, timezone
 from typing import Any, Optional
 
 import requests
 from flask import current_app as app
-from singleton_decorator import singleton
+from unstract.platform_service.constants import FeatureFlag
 from unstract.platform_service.env import Env
 from unstract.platform_service.utils import format_float_positional
+from unstract.sdk.exceptions import FileStorageError
+from unstract.sdk.file_storage import FileStorageProvider, PermanentFileStorage
+
+from unstract.flags.feature_flag import check_feature_flag_status
 
 
-@singleton
```

Comment:
Removing singleton here. With SIngleton, the TTL check will only happen when container restarts and the 7 days TTL config may not be adhered to.
Anyway, the plan is to create a separate cron job for the json file pull and updation. Once that is in place, the TTL/file expiry  logic can be safely removed.
@chandrasekharan-zipstack

Reply:
@gaya3-zipstack will the cron job to pull and update be run within the container? If not, we could just subscribe to changes in that JSON and try to make use of some action to update our code / push the file

Reply:
@chandrasekharan-zipstack Also removing the singleton-decorator dependency. Looks like this was the only place where we use singleton.

Reply:
I hope this is being tracked somewhere, please help create a ticket if not @gaya3-zipstack and feel free to resolve this conversation
cc: @hari-kuriakose